src/crewai/agent.py

```diff
@@ -203,9 +204,9 @@ def __create_agent_executor(self) -> None:
         }
 
         if self._rpm_controller:
-            executor_args[
-                "request_within_rpm_limit"
-            ] = self._rpm_controller.check_or_wait
+            executor_args["request_within_rpm_limit"] = (
```

Comment:
I've been fighting with this for a long time. Github's `black` says this is the correct form, while VSCode insist on the the former way.